src/transformers/pipelines.py

```diff
@@ -96,19 +97,50 @@ class DefaultArgumentHandler(ArgumentHandler):
     Default varargs argument parser handling parameters for each Pipeline
     """
 
-    def __call__(self, *args, **kwargs):
-        if "X" in kwargs:
-            return kwargs["X"]
-        elif "data" in kwargs:
-            return kwargs["data"]
-        elif len(args) == 1:
-            if isinstance(args[0], list):
-                return args[0]
-            else:
+    @staticmethod
+    def handle_kwargs(kwargs: Dict) -> List:
+        if len(kwargs) == 1:
+            output = list(kwargs.values())
+        else:
+            output = list(chain(kwargs.values()))
```

Comment:
Adding the extra `*` will lead to unwanted behaviour in the following case: 

Imagine you have (quite surprising but not impossible) input such as `{"data: "some str", "X": "some other str"}` it will explode both `str` and convert them to a flattened list of char. 

The actual benefits of chain is to concatenate all the possible kwargs and conserving only the `values` and not the `key`